### ISSUE TITLE
Google Fonts Issue

### ISSUE THREAD

**Issue by dev-opos** [validator/validator#684]
It shows an error when linking two, or more fonts from google. When adding this line to html it shows an error but it works and i think it is correct.

Error: Bad value https://fonts.googleapis.com/css?family=Roboto+Condensed:300|Cinzel+Decorative:700 for attribute href on element link: Illegal character in query: | is not allowed.

**Comment by LeeKowalkowski** [comment]
I think you're supposed to escape the `|` character with `%7C`, i.e. https://fonts.googleapis.com/css?family=Roboto+Condensed:300%7CCinzel+Decorative:700 

https://developers.google.com/maps/documentation/urls/url-encoding

**Comment by sideshowbarker** [comment]
> I think you're supposed to escape the `|` character with `%7C`, i.e. https://fonts.googleapis.com/css?family=Roboto+Condensed:300%7CCinzel+Decorative:700 

Right. Per the current URL spec, `|` isn’t a valid character in URLs. I must be percent-encoded as `%7C`.

**Comment by nschonni** [comment]
Duplicate of #619
